mods/content/xenobiology/slime/life.dm

```diff
@@ -65,12 +65,10 @@
 
 /mob/living/slime/fluid_act(datum/reagents/fluids)
 	. = ..()
-	if(stat == DEAD)
-		var/obj/effect/fluid/F = locate() in loc
-		if(F && F.reagents?.total_volume >= FLUID_SHALLOW)
-			F.reagents.add_reagent(/decl/material/liquid/slimejelly, (is_adult ? rand(30, 40) : rand(10, 30)))
-			visible_message(SPAN_DANGER("\The [src] melts away...")) // Slimes are water soluble.
-			qdel(src)
+	if(stat == DEAD && loc?.reagents?.total_volume >= FLUID_SHALLOW)
```

Comment:
may be worth changing this to check the volume of water in it specifically, or use some other value. otherwise this will cause a cascade of slime jelly puddles dissolving other slimes